webclient/ez/migrations/0005_alter_externizdroj_stav.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.2.3 on 2023-09-15 15:47
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("ez", "0004_externizdrojsekvence_and_more"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="externizdroj",
+            name="stav",
+            field=models.SmallIntegerField(
+                choices=[(1, "EZ1 - zapsaný"), (2, "Odeslaný"), (3, "EZ2 - potvrzený")]
```

Comment:
@jnihnat Toto mi přijde nějaké zmatečné mělo by případně být:
```python
choices=[(1, "EZ1 - zapsaný"), (2, "EZ2 - odeslaný"), (3, "EZ3 - potvrzený")]
``` 
Ale mělo by se to ale spíše nějak vázat na (?):
https://github.com/ARUP-CAS/aiscr-webamcr/blob/2e8d36505ca430fc10a04ef0a175b357762e42ed/webclient/core/constants.py#L37

Reply:
Psal jsem to uz Jirkovi ze se mi nejak objevili migrace navic ci o tom neco nevi. Tedka jsem jeste na to koukl a mozna i vim odkud vane vitr. Zitra na to jeste kouknu.

Reply:
Tohle je opravene a podobna chyba bola v pas. Budeme si muset dat pozor do budoucna, ta cast s labelem se bere totiz z prekladu a ten byl spatny.